core/src/trezor/ui/layouts/common.py

```diff
@@ -19,24 +19,26 @@ def report(self, value: int, description: str | None = None) -> None: ...
 
 
 async def button_request(
-    br_type: str,
+    br_name: str,
     code: ButtonRequestType = ButtonRequestType.Other,
     pages: int | None = None,
 ) -> None:
     if __debug__:
-        log.debug(__name__, "ButtonRequest.type=%s", br_type)
+        log.debug(__name__, "ButtonRequest.type=%s", br_name)
```

Comment:
please drop this whole `if __debug__`, its only purpose was so that the br_name is not lost. now that we actually send it out, it's not necessary anymore

Reply:
https://github.com/trezor/trezor-firmware/pull/3998/commits/88a99d7b3837c5e7d7074cd0a9fc21b85f5bff40#diff-cf8678b9cabe8805b6f4884da5b4d2acee0da7f15471d6d9026441e46e9cf09eL26